opentelemetry/proto/logs/v1/logs.proto

```diff
@@ -32,6 +32,11 @@ message ResourceLogs {
 
   // A list of InstrumentationLibraryLogs that originate from a resource.
   repeated InstrumentationLibraryLogs instrumentation_library_logs = 2;
+
+  // This schema_url applies to the "resource" field and to all logs in the
+  // "instrumentation_library_logs" except the logs which have a schema_url
+  // specified in the nested InstrumentationLibraryLogs message.
+  string schema_url = 3;
```

Comment:
Can we start without this optimization? How is this set in the API/SDK?

Reply:
See application-wide schema URL in SDK: https://github.com/open-telemetry/opentelemetry-specification/pull/1666/files

Reply:
This is not an optimization. It is _required_ to describe the attributes of the Resource.

Reply:
After discussing with @bogdandrutu and putting some more though into this I am going to make a change here. The `schema_url` here will describe the Resource, but will not have any effect on the spans (the overriding logic that it has in the current form). This is unnecessary complication, it saves maybe a tiny bit of data in a request, but creates confusion.
I am going to delete this "overriding" logic and simplify it. Will post a commit in a bit.

Reply:
OK, see the new commit https://github.com/open-telemetry/opentelemetry-proto/commit/dd92c019a527e55d1f7b926ed4618de7ef4a9369 that does this. Only the comments are changed (which changes the semantic of this field).
I believe this is simpler and easier to understand.